packages/react-component-library/src/components/ContextMenu/ContextMenu.tsx

```diff
@@ -5,17 +5,38 @@ import { useClickMenu, ClickType } from '../../hooks/useClickMenu'
 import { StyledContextMenu } from './partials/StyledContextMenu'
 
 interface ContextMenuProps extends ComponentWithClass {
-  attachedToRef?: React.RefObject<HTMLElement>
+  /**
```

Comment:
Did you find this interface confusing without the comments? 

Would you maybe like to see more of these elsewhere?

Reply:
It's fine without them, but having comments across the board helps remove ambiguity and it's just a good thing to do :)